Content.Shared/_Harmony/CCVars/HCCVars.cs

```diff
@@ -0,0 +1,23 @@
+using Robust.Shared.Configuration;
+
+namespace Content.Shared._Harmony.CCVars;
+
+/// <summary>
+/// Harmony-specific cvars.
+/// </summary>
+[CVarDefs]
+public sealed class HCCVars
+{
+    /// <summary>
+    /// Modifies suicide command to ghost without killing the entity.
+    /// </summary>
+    /// Frontier: Disabled the anti suicide code pending discussion about wanting it
+   /// public static readonly CVarDef<bool> DisableSuicide =
+   ///     CVarDef.Create("ic.disable_suicide", false, CVar.SERVER);
+   /// end Frontier
+    /// <summary>
+    /// Allows server hosters to turn the queue on and off
+    /// </summary>
+    public static readonly CVarDef<bool> EnableQueue =
+        CVarDef.Create("queue.enable", false, CVar.SERVER);
```

Comment:
If its giving issues with allowing you to change the value at run-time, try changing these paramaters. They will dictate who and what permission level is allowed to change this cvar in-game. I would suggest keeping it under a higher permission than what a regular game admin might have.